pkg/registry/registry.go

```diff
@@ -40,6 +44,10 @@ func New(c Config) (*Registry, error) {
 }
 
 func (r *Registry) Login(ctx context.Context, user, password string) error {
+	if r.lastLogin != nil {
+		return nil
+	}
```

Comment:
I feel that's going to be weird because we'll still print messages about logging in. Does it make sense to do that in the runner instead?

Reply:
Moved to runner.